src/components/app/MenuButtons/index.js

```diff
@@ -18,10 +18,13 @@ import {
 } from 'firefox-profiler/selectors/profile';
 import { getDataSource } from 'firefox-profiler/selectors/url-state';
 import { getIsNewlyPublished } from 'firefox-profiler/selectors/app';
-import { MenuButtonsMetaInfo } from 'firefox-profiler/components/app/MenuButtons/MetaInfo';
+
+/* Note: the order of import is important, from most general to most specific,
```

Comment:
yeah, I'm not sure of the right solution.
This is more ArrowPanel.css (that's imported by ArrowPanel imported by ButtonWithPanel).

Another idea: in the components that depend on another CSS (sort of an inheritance) we could import that CSS there as well, so that we force the order. We could force all ArrowPanel-children components to import some other component (`ArrowPanelChild` ?) through Flow, and that one would import the dependent CSS, that could also work. A bit more verbose...

For now I think the comment is good enough. But it's not the first place where I've seen similar issues (and added similar comments).